src/components/Breadcrumbs/index.tsx

```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
     component: any;
     hrefProperty: string;
   };
+  backgroundColor?: 'light' | 'dark';
   links: ILink[];
 }
 
+let background = 'light';
+const DarkMode =
```

Comment:
do the dark mode background handling inside the stories file by just adding another ComponentStory